DESCRIPTION

```diff
@@ -44,3 +46,4 @@ Config/testthat/parallel: true
 Config/testthat/start-first: vs-es, scan, vs-operators, weakref, watts.strogatz.game
 RoxygenNote: 7.2.3
 Roxygen: list(markdown = TRUE)
+VignetteBuilder: knitr
```

Comment:
I ran `usethis::use_vignette("igraph")` to create the vignette. It added the necessary dependencies (rmarkdown, knitr) to `DESCRIPTION`.